inlineplz/interfaces/github.py

```diff
@@ -67,12 +68,12 @@ def is_duplicate(self, message, position):
         return False
 
     @staticmethod
-    def format_message(message):
+    def format_message(message, prefix):
         if not message.comments:
             return ''
         if len(message.comments) > 1:
             return (
-                '```\n' +
+                '```\n' + prefix + '\n' +
```

Comment:
i think you might also need to handle the 3rd case of a comment of length 1 (or just get rid of that case entirely)